File: py/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris): # inherit dari class Aksesoris
    # atribut
    __untuk = ""
    __size = ""
    __merk = ""
    
    # constructor
    def __init__(self, untuk="", size="", merk=""):
        self.__untuk = untuk
        self.__size = size
        self.__merk = merk
        
    # getter
    def getUntuk(self): return self.__untuk
    def getSize(self): return self.__size
    def getMerk(self): return self.__merk
    
    # setter
    def setUntuk(self, untuk): self.__untuk = untuk
    def setSize(self, size): self.__size = size
    def setMerk(self, merk): self.__merk = merk

File: java/Baju.java
public class Baju extends Aksesoris{ // inherit dari class Aksesoris
    // atribut
    private String untuk;
    private String size;
    private String merk;

    // constructor

    public Baju(){

    }
    
    public Baju(String untuk, String size, String merk){
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    // getter
    public String getUntuk() { return this.untuk; }
    public String getSize() { return this.size; }
    public String getMerk() { return this.merk; }

    // setter
    public void setUntuk(String untuk) { this.untuk = untuk; }
    public void setSize(String size) { this.size = size; }
    public void setMerk(String merk) { this.merk = merk; }




}